src/ecs/component_panels.rs

```diff
@@ -495,3 +496,79 @@ impl ComponentPanel for Ruler {
         });
     }
 }
+
+impl ComponentPanel for Sphere {
+    fn inspector_name() -> &'static str {
+        "Sphere"
+    }
+
+    fn inspector_icon() -> char {
+        ICON_SPHERE
+    }
+
+    fn has_inspector_ui() -> bool {
+        true
+    }
+
+    fn show_inspector_ui(&mut self, ui: &mut egui::Ui, resources: &Resources) {
+        let camera = resources.get::<FpsCamera>().unwrap();
+        ui.horizontal(|ui| {
+            input_float3!(ui, format!("{ICON_ALPHA_A_BOX} Center"), &mut self.center);
+            if ui
+                .button(ICON_CAMERA_CONTROL.to_string())
+                .on_hover_text("Set position to camera")
+                .clicked()
+            {
+                self.center = camera.position;
+            }
+        });
+
+        ui.horizontal(|ui| {
+            ui.strong("Radius");
+            ui.add(
+                egui::DragValue::new(&mut self.radius)
+                    .speed(0.1)
+                    .clamp_range(0f32..=f32::INFINITY)
+                    .min_decimals(2)
+                    .max_decimals(2)
+                    .suffix(" m"),
+            );
+            if ui
+                .button(ICON_CAMERA_CONTROL.to_string())
```

Comment:
Icon should probably be ICON_RADIUS_OUTLINE, seems more appropriate

Reply:
For future reference, alkahest uses the pictogrammers material design icon library
https://pictogrammers.com/library/mdi/